docs/assemblies/deploying.adoc

```diff
@@ -688,3 +688,34 @@ $ oc get openstackdataplanenodeset openstack-edpm-ipam -o json | jq '.status.con
   "type": "NodeSetDNSDataReady"
 }
 ----
+
+=== Dataplane deployment Pods
+
+
+During reconciliation of OpenStackDataPlaneDeployment resources,
+OpenStackAnsibleEE resources, marked with label `osdpd=<OpenStackDataPlaneDeployment.Name>,
+are created to manage the ansible executions that configure the EDPM nodes.
+The OpenStackAnsibleEE resources create Kubernetes Jobs resources, which in turn create Kubernetes Pods that are executing ansible.
+
+Checking the logs of these Jobs and Pods can show ansible failures and help diagnose issues.
+
+Pods launched during deployement throught OpenStackAnsiblEEE managed jobs,
+further referred to as pods, are are marked with label `osdpd=<OpenStackDataPlaneDeployment.Name>`
+and checked for failures.
+
+While the recovery of jobs is an automated process,
+additional information can be useful for diagnosing the issue.
+
+Failure of individual pods is logged on info level in format:
+
+[,console]
+----
+ `"openstack job <POD_NAME> failed due to <FAILURE_REASON> with message: <FAILURE_MSG>"`
```

Comment:
need to update to match the message from the code

Reply:
Because of recent documentation changes the doc doesn't exist anymore.

Reply:
But I've changed the message anyway.